src/recensio/plone/behaviors/authors.py

```diff
@@ -43,3 +44,10 @@ def authors(self):
     @authors.setter
     def authors(self, value):
         self.context.authors = value
+
+    def get_formatted_authors(self):
+        authors_list = [
+            getFormatter(" ")(author.to_object.firstname, author.to_object.lastname)
+            for author in self.context.authors
+        ]
```

Comment:
This looks safer,  does not call twice `to_object` for every iteration and does initialize `getFormatter(" ")` only once:
```suggestion
        formatter = getFormatter(" ")
        authors_list = []
        for item in self.context.authors:
            obj = item.to_object
            if obj:
                authors_list.append(formatter(obj.firstname, obj.lastname))
```